lib/pitchfork/http_server.rb

```diff
@@ -896,7 +896,7 @@ def nuke_listeners!(readers)
 
     # gets rid of stuff the worker has no business keeping track of
     # to free some resources and drops all sig handlers.
-    # traps for USR2, and HUP may be set in the after_fork Proc
+    # traps for USR2, and HUP may be set in the after_worker_fork/after_mold_fork Procs
```

Comment:
I am unsure if both hook are relevant to mention in this context.

Reply:
Doesn't hurt.